.default.env

```diff
@@ -1,2 +0,0 @@
-ENVIRONMENT=development
-SWARM_EXTENSION_ID=oncdlaccbflglnchidicplbcikejjcid
```

Comment:
Why are these properties deleted? We should keep them as reference, to easily copy/paste when running the app for first time.